src/lib/transform.js

```diff
@@ -150,6 +152,24 @@ const arTransformers = [
   'lastSaved',
 ];
 
+function validateCsvRows(csvRows) {
```

Comment:
The only concern I have is the ordering of the goal objectives as we discussed (minor).

If I enter obj 1, obj 2, obj 3 I get out obj 3, obj 2, obj 1 but there might not be a better way right now.